sdk/synapse/Microsoft.Azure.Management.Synapse/src/Microsoft.Azure.Management.Synapse.csproj

```diff
@@ -7,11 +7,11 @@
 		<PackageId>Microsoft.Azure.Management.Synapse</PackageId>
 		<Description>Azure Synapse Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Synapse</AssemblyName>
-		<Version>2.1.0-preview</Version>
+		<Version>2.2.0-preview</Version>
 		<PackageTags>Microsoft Azure Synapse Management;Synapse;Synapse Management</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
-			- SDK now consumes 2021-04-01-preview API version
+			- SDK now consumes 2021-06-01 API version
```

Comment:
I saw the API versions are actually the combination of `2021-06-01` and `2021-06-01-preview`. Better to mention it to end users.